test/extended/cni_vendor_test.sh

```diff
@@ -2,10 +2,15 @@
 
 source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
 
-# Set this to false if namespaces are not isolated
-export OPENSHIFT_NETWORK_ISOLATION="${OPENSHIFT_NETWORK_ISOLATION:-true}"
+# Set this to false if the plugin does not implement NetworkPolicy
+export NETWORKING_E2E_NETWORKPOLICY="${NETWORKING_E2E_NETWORKPOLICY:-true}"
+
+# Set this to true if the plugin implements isolation in the same manner as
+# redhat/openshift-ovs-multitenant
+export NETWORKING_E2E_ISOLATION="${NETWORKING_E2E_ISOLATION:-false}"
```

Comment:
Sorry, partly explained in the commit message but not the code; what I remember from discussions is that we don't expect third parties to implement ovs-multitenant-style isolation directly, we expect them to just implement NetworkPolicy, and we'll eventually move to a system where "oc new-project" or whatever creates appropriate policies in each namespace if the cluster is supposed to be isolated-by-default.